### PR TITLE
CHANGELOG: update changelog for 1.6.0 release

### PR DESCRIPTION
**Pull request by liangwen12year**
Signed-off-by: Wen Liang <liangwen12year@gmail.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
+--------------------
+
+### New features
+
+- Support routing tables in static routes
+
+### Bug fixes
+
+- Fix setting DNS search settings only for enabled IP families
+- Fix reloading the configuration for nm provider on NetworkManager 1.18
```
Comment (ffmancera) [inline CHANGELOG.md:14]: I think the right description would be "Fix switching providers on NetworkManager 1.18". What do you think?

**Review by ffmancera — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (tyll) [reply CHANGELOG.md:14]: Technically, it is not only about switching the provider but also using the role the first time with the NM provider on a system that used network-scripts in the past and NM_CONTROLLED=no.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,18 @@
...
+[1.6.0] - 2022-02-02
+--------------------
+
+### New features
+
+- Support routing tables in static routes
+
+### Bug fixes
+
+- Fix setting DNS search settings only for enabled IP families
```
Comment (tyll) [inline CHANGELOG.md:13]:
```suggestion
- Fix setting DNS search settings if only one IP family is enabled
```

File: CHANGELOG.md (hunk 1 above)
Comment (tyll) [reply CHANGELOG.md:14]:
```suggestion
- Fix switching from initscripts to NetworkManager 1.18
```

Comment (liangwen12year) [reply CHANGELOG.md]: I updated, thanks.

**Review by ffmancera — APPROVED** [review]
LGTM!

**Pull request merged**